docs/components/typography.md

```diff
@@ -0,0 +1,80 @@
+<script setup>
+import { SwText, FONT_FAMILY, FONT_WEIGHTS } from '@swimm/ui';
+</script>
+<style scoped>   
+.theme-default-content h1,
+.theme-default-content h2,
+.theme-default-content h3,
+.theme-default-content h4,
+.theme-default-content h5,
+.theme-default-content h6 {
+    margin: 16px 0;
+    padding: 0;
+    border: none;
+}
+</style>
```

Comment:
I think we should move it to the vuepress stylesheet and have our entire typography all over, will it work?

Reply:
Not sure